funnel/templates/notifications/comment_received_email.html.jinja2

```diff
@@ -1,19 +1,35 @@
 {%- extends "notifications/layout_email.html.jinja2" -%}
+{%- from "notifications/macros_email.html.jinja2" import hero_image, cta_button -%}
 {%- block content -%}
 
-  {%- if view.notification.document_type == 'project' -%}
-    <h2><a href="{{ view.document.url_for(_external=true, **view.tracking_tags()) }}">{{ view.document.joined_title }}</a></h2>
-  {%- elif view.notification.document_type == 'proposal' -%}
-    <h2><a href="{{ view.document.url_for(_external=true, **view.tracking_tags()) }}">{{ view.document.title }}</a></h2>
-  {%- elif view.notification.document_type == 'comment' -%}
-    <p>{% trans %}You wrote:{% endtrans %}</p>
-    <blockquote type="cite">{{ view.document.message }}</blockquote>
-  {%- endif %}
+    {# Hero image centered : BEGIN #}
+    {{ hero_image("https://images.hasgeek.com/embed/file/eb253bce19fd41ddb5ec84b225bd4dc2?size=196x165") }}
+    {# Hero image centered : END #}
 
-  <p>{{ view.activity_html() }}</p>
+    {# Email body : BEGIN #}
+    <tr align="center">
+        <td class="text-container">
+            <h1>New comment!</h1>
+            {%- if view.notification.document_type == 'project' -%}
+              <h3><a href="{{ view.document.url_for(_external=true, **view.tracking_tags()) }}">{{ view.document.joined_title }}</a></h3>
+            {%- elif view.notification.document_type == 'proposal' -%}
+              <h3><a href="{{ view.document.url_for(_external=true, **view.tracking_tags()) }}">{{ view.document.title }}</a></h3>
+            {%- elif view.notification.document_type == 'comment' -%}
+              <p>{% trans %}You wrote:{% endtrans %}</p>
+              <blockquote type="cite">{{ view.document.message }}</blockquote>
+            {%- endif %}
 
-  <blockquote type="cite">{{ view.comment.message }}</blockquote>
+            <p>{{ view.activity_html() }}</p>
 
-  <p class="btn-bar"><a class="btn" href="{{ view.comment.url_for(_external=true, **view.tracking_tags()) }}">{% trans %}View comment{% endtrans %}</a></p>
+            <blockquote type="cite">{{ view.comment.message }}</blockquote>
+        </td>
+    </tr>
+    <br>
+
+    {# Button : BEGIN #}
+    {{ cta_button(view.comment.url_for(_external=true, **view.tracking_tags()), "View comment" )}}
```

Comment:
Here the text should be `gettext("View comment")`

Reply:
Fixed this.